cvat-ui/src/components/header/settings-modal/multiple-shortcuts-display.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useRef, useState, useEffect } from 'react';
+import { Select, Modal } from 'antd/lib';
+import { conflict, conflictDetector } from 'utils/conflict-detector';
+import { ShortcutScope } from 'utils/enums';
+import { KeyMapItem } from 'utils/mousetrap-react';
+import { registerComponentShortcuts } from 'actions/shortcuts-actions';
+import { getKeyfromCode } from 'utils/key-code-mapper';
+import { isEqual } from 'lodash';
+
+interface Props {
+    id: string;
+    item: { sequences: string[]; name: string; description: string; scope: string };
+    keyMap: Record<string, KeyMapItem>
+    onKeySequenceUpdate: (keyMapId: string, updatedSequence: string[]) => void;
+}
+
+function MultipleShortcutsDisplay(props: Props): JSX.Element {
+    const {
+        id,
+        item,
+        keyMap,
+        onKeySequenceUpdate,
+    } = props;
+    const { sequences } = item;
+    const selectRef = useRef<any>(null);
+    const [focus, setFocus] = useState(false);
+    const [pressedKeys, setPressedKeys] = useState<string[][]>([[]]);
+    const [currentIdx, setCurrentIdx] = useState<number>(0);
+    const [timer, setTimer] = useState<NodeJS.Timeout | null>(null);
+
+    useEffect(() => () => {
+        if (timer) clearTimeout(timer);
+    }, [timer]);
+
+    function unsetExistingShortcut(
+        conflictingShortcuts: Record<string, KeyMapItem>,
+        updatedSequence: string[],
+        shortcut: Record<string, KeyMapItem>): void {
+        const updatedShortcuts: Record<string, KeyMapItem> = {};
+        for (const [key, value] of Object.entries(conflictingShortcuts)) {
+            const currentItem: Record<string, KeyMapItem> = {
+                [key]: value,
+            };
```

Comment:
Why not do the same thing simpler? 

```suggestion
        for (const key of Object.keys(conflictingShortcuts)) {
            const conflictingItem: KeyMapItem = conflictingShortcuts[key];
```